R/plot_section.R

```diff
@@ -0,0 +1,153 @@
+#' Generate stratigraphic section plot
+#'
+#' A function to plot the stratigraphic ranges of fossil taxa from occurrence
+#' data.
+#'
+#' @param occdf \code{dataframe}. A dataframe of fossil occurrences containing
+#' at least two columns: names of taxa, and their stratigraphic position
+#' (see `name` and `level` arguments).
+#' @param name \code{character}. The name of the column you wish to be treated
+#' as the input names, e.g. "genus" (default).
```

Comment:
```suggestion
#' as the input names, e.g. "genus" (default).
```
"taxon" seems to be the current default?